output_writer.cpp

```diff
@@ -0,0 +1,59 @@
+#include "output_writer.hpp"
+#include "cpp_tstring.hpp"
+#include "fizz_buzz.hpp"
+#include <vector>
+#include <algorithm>
+#include "console.hpp"
+#include <tchar.h>
+#include <algorithm>
+
+void BaseOutputWriter::writeOne(unsigned long long int number, FizzBuzzResult result) {
+	tstring str;
+	if (number == 0) str = TEXT("0");
+	else while (number != 0) {
+		str.push_back((TCHAR)((number % 10) + TEXT('0')));
+		number /= 10;
+	}
+	std::reverse(str.begin(), str.end());
+	str.append(TEXT(":"));
+	if ((result & FizzBuzzResult::FIZZ) != FizzBuzzResult::UNKNOWN) str.append(TEXT("Fizz"));
```

Comment:
with the ostream operator<< I mentioned earlier, this can use ostringstream. something like:
```cpp
std::basic_ostringstream<TCHAR> os;
os << result;
str.append(os.str());
```

Reply:
Commit 8c00abc09ff3c1a42f745f9e5d3b41e44fb311f4